stactools_landsat/stactools/landsat/utils.py

```diff
@@ -0,0 +1,124 @@
+import datetime
+
+import dateutil
+import rasterio
+from pystac import Item, Link
+from rasterio import RasterioIOError
+from shapely.geometry import box, mapping, shape
+
+
+def _parse_date(in_date: str) -> datetime.datetime:
+    """
+    Try to parse a date and return it as a datetime object with no timezone
+    """
+    dt = dateutil.parser.parse(in_date)
+    return dt.replace(tzinfo=datetime.timezone.utc)
+
+
+def transform_mtl_to_item(metadata: dict) -> dict:
+    """
+    Handle USGS MTL as a dict and return a STAC dict.
+
+    This isn't implemented, because all the info we need
```

Comment:
Confused by this comment - do you mean it's not used?

Reply:
This was only started and not completed, because the MTL is missing a lot of information...

I don't think using the MTL is a good idea at all!